app/services/heading_service/cached_heading_service.rb

```diff
@@ -61,19 +61,15 @@ def filter_commodity_relations
           has_valid_dates(measure, :effective_start_date, :effective_end_date)
         end
 
-        commodity.overview_measures = OverviewMeasurePresenter.new(commodity.overview_measures, commodity).validate!
-
-        commodity.overview_measure_ids = commodity.overview_measures.map do |measure|
-          measure.measure_sid
-        end
+        commodity.overview_measure_ids = commodity.overview_measures.map(&:measure_sid)
 
         commodity.goods_nomenclature_indents.keep_if do |ident|
           has_valid_dates(ident)
         end
 
-        indent = commodity.goods_nomenclature_indents.sort_by do |ident|
+        indent = commodity.goods_nomenclature_indents.sort_by { |ident|
           Date.parse ident.validity_start_date
-        end.last
+        }.last
```

Comment:
This is not inline, the convention is to use do/end block right?
or you wanted to inline it, which could be fine

Reply:
I used rubocop to do this, actually. It wants {} over multilines when there is a method cool on the block result :man_shrugging: